Jenkinsfile

```diff
@@ -30,7 +30,7 @@ else if (env.CHANGE_ID) {
     /* When handling pull requests, ensure everything is denoted by the pull
      * request
      */
-    tfPrefix = "pr${env.BUILD_NUMBER}"
+    tfPrefix = "pr${env.CHANGE_ID}${env.BUILD_NUMBER}"
```

Comment:
What is 'env.CHANGED_ID' for?
I am looking for definition [here](https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-JenkinsSetEnvironmentVariables)

Are we sure that CHANGE_ID will never contain special charachter like '-' ?